pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -603,14 +609,54 @@ message ExecRequest {
     // Whether use tty
     optional bool tty = 3;
     // Streaming stdin
-    optional bytes stdin = 4;
+    optional bool stdin = 4;
 }
 
 message ExecResponse {
-    // Streaming stdout
+    // The fully qualified URL of the exec streaming server
+    optional string url = 1;
+}
+
+message ExecSyncRequest {
```

Comment:
Ack. I was following the example of our other methods, so we can change these new methods too if we need to change them all.